gradio/blocks.py

```diff
@@ -1362,10 +1362,10 @@ def handle_streaming_outputs(
                 if run not in self.pending_streams[session_hash]:
                     self.pending_streams[session_hash][run] = defaultdict(list)
                 self.pending_streams[session_hash][run][output_id].append(stream)
-                data[i] = {
-                    "name": f"{session_hash}/{run}/{output_id}",
-                    "is_stream": True,
-                }
+                if data[i]:
+                    data[i]["is_file"] = False
+                    data[i]["name"] = f"{session_hash}/{run}/{output_id}"
```

Comment:
@aliabid94 Thoughts on including the `stream` endpoint name in the route so that it doesn't have to be hardcoded in the client?